var/spack/repos/builtin/packages/pcsclite/package.py

```diff
@@ -0,0 +1,54 @@
+# Copyright 2013-2022 Lawrence Livermore National Security, LLC and other
+# Spack Project Developers. See the top-level COPYRIGHT file for details.
+#
+# SPDX-License-Identifier: (Apache-2.0 OR MIT)
+
+from spack.package import *
+
+
+class Pcsclite(AutotoolsPackage):
+    """PCSC lite project
+
+    Middleware to access a smart card using SCard API (PC/SC)."""
+
+    homepage = "https://pcsclite.apdu.fr"
+    url = "https://pcsclite.apdu.fr/files/pcsc-lite-1.9.8.tar.bz2"
+    git = "https://salsa.debian.org/rousseau/PCSC.git"
+
+    maintainers = ["cessenat"]
+
+    version("master", branch="master")
+    version("1.9.8", sha256="502d80c557ecbee285eb99fe8703eeb667bcfe067577467b50efe3420d1b2289")
+
+    # no libudev/systemd package currently in spack
+    variant("libudev", default=False, description="Build with libudev")
+
+    depends_on("flex", type="build")
+    depends_on("libusb")
+
+    depends_on("autoconf", type="build")
+    depends_on("autoconf-archive", type="build")
+    depends_on("automake", type="build")
+    depends_on("libtool", type="build")
+    depends_on("m4", type="build")
+    depends_on("pkgconfig", type="build")
+
+    phases = ["bootstrap", "configure", "build", "install"]
```

Comment:
Can we stick with the `autoreconf` name, and override that?

Reply:
Well, no : without the specific phases for the 'master' release I get errors
`aclocal: error: couldn't open directory 'm4': No such file or directory`
So, this is the reason for this heavy workaround
[spack-build-out.txt](https://github.com/spack/spack/files/9382592/spack-build-out.txt)
[spack-build-env.txt](https://github.com/spack/spack/files/9382593/spack-build-env.txt)

Reply:
Sorry I wasn't clear enough. Why does it matter to call the phase `bootstrap`? You can simply:
```python
def autoreconf(self, spec, prefix):
    pass

@when("@master")
def autoreconf(self, spec, prefix):
    bootstrap = Executable("./bootstrap")
    bootstrap()
```
and remove both `phases` (which are not meant to be overridden) and `force_autoreconf` since you don't use the default autoreconf method.

Reply:
Yes you are right. Commited accordingly. Thanks.